### ISSUE TITLE
Allow --no-sandbox parameter

### ISSUE THREAD

**Issue by mynameisnotbruce** [igorkasyanchuk/rails_pdf#10]
I've been trying to get this to work on Heroku, and one of the issues I encountered is that puppeteer complains about unavailable sandboxes.

The quick fix was to just include `--no-sandbox` at the end of the `command` in `renderer.rb`. But perhaps it would make more sense to make this optional e.g. via a config setting or something?

I'm also not sure if there are adverse effects by switching off sandboxing, e.g. security concerns, and if they should be mitigated somehow.

**Comment by igorkasyanchuk** [comment]
@mynameisnotbruce if you add this option does it work on Heroku? PDF generation I mean with your branch.

**Comment by mynameisnotbruce** [comment]
Yes, adding `--no-sandbox` to the command makes it possible to generate the PDFs. But there is a warning in the console and I guess it comes with security downsides, so like I said, making this an optional setting somehow might be the best approach here.
